Index.html

```diff
@@ -92,6 +92,128 @@ <h2 class="jobtitle">Software Developer</h2>
+                          no accounts or sign-ups required. 
+                          has been the industry's standard dummy text ever since the 1500s, 
+                          when an unknown printer took a standard dummy text.
+                      </p>
+                      <ul class="flex-item navlang-flex">
+                          <li><a href="#" class="subflex-item artag">CSS</a></li>
+                          <li><img src="image/css-divider.png" alt="divider image" class="subflex-item"></li>
+                          <li><a href="#" class="subflex-item artag">html</a></li>
+                          <li><img src="image/css-divider.png" alt="divider image" class="subflex-item"></li>
+                          <li><a href="#" class="subflex-item artag">bootstrap</a></li>
+                          <li><img src="image/css-divider.png" alt="divider image" class="subflex-item"></li>
+                          <li><a href="#" class="subflex-item artag">Ruby</a></li>
+                          </ul>
+                      <ul class="seep"><li><a href="#" class="subflex-item seeproject">See Project</a></li></ul>
+                  </div>
+                  
+              </section>
+
+              <section class="display-project4">
+                  <div class="placeholder-div">
+                      <p><img src="image/Img-Placeholder.png" alt="image of project" class="ph-image"></p>
+                      
+
+                  </div>
+                  <div class="project-text-nav">
+                      <h2 class="hh2">Multi-Post Story</h2>
+                      <p class="flex-item holdtext">
+                          A daily selection of privately personalized reads; 
+                          no accounts or sign-ups required. 
+                          has been the industry's standard dummy text ever since the 1500s, 
+                          when an unknown printer took a standard dummy text.
+                      </p>
+                      <ul class="flex-item navlang-flex">
+                          <li><a href="#" class="subflex-item artag">CSS</a></li>
+                          <li><img src="image/css-divider.png" alt="divider image" class="subflex-item"></li>
+                          <li><a href="#" class="subflex-item artag">html</a></li>
+                          <li><img src="image/css-divider.png" alt="divider image" class="subflex-item"></li>
+                          <li><a href="#" class="subflex-item artag">bootstrap</a></li>
+                          <li><img src="image/css-divider.png" alt="divider image" class="subflex-item"></li>
+                          <li><a href="#" class="subflex-item artag">Ruby</a></li>
+                          </ul>
+                      <ul class="seep"><li><a href="#" class="subflex-item seeproject">See Project</a></li></ul>
+                  </div>
+                  
+              </section>
+
+          </div>
+
+      </article>
```

Comment:
- [ ] I should mention the hard work you put into practicing clean code and using semantic tags throughout your code 👌🏽 shows that you care about all your site users and search engines. I must say you did a wonderful job ✨ and keep up the good work 👏🏽.